spikeinterface/extractors/neoextractors/openephys.py

```diff
@@ -63,22 +65,30 @@ def __init__(self, folder_path, stream_id=None):
         neo_kwargs = {'dirname': folder_path}
         NeoBaseRecordingExtractor.__init__(self, stream_id=stream_id, **neo_kwargs)
 
+        if hasattr(pi, "read_openephys"):
+            try:
```

Comment:
I would prefer to use the try/except for this.
Could we make in probeinterface side a function that test if the probe exists in the folder ?
also `pi.read_openephys(folder_path)` could return None when no probe.